app/src/components/v-form/form-field-interface.vue

```diff
@@ -77,14 +80,30 @@ export default defineComponent({
 			type: Boolean,
 			default: false,
 		},
+		values: {
+			type: Object,
+			default: null,
+		},
 	},
 	emits: ['update:modelValue', 'setFieldValue'],
-	setup(props) {
+	setup(props, { emit }) {
 		const { t } = useI18n();
 
 		const interfaceExists = computed(() => !!getInterface(props.field?.meta?.interface || 'input'));
 
-		return { t, interfaceExists, getDefaultInterfaceForType };
+		const api = useApi();
+		const { useCollectionsStore, useFieldsStore } = useStores();
+		const collectionsStore = useCollectionsStore();
+		const fieldsStore = useFieldsStore();
+
+		let context = {
```

Comment:
Creating a context object here isn't really saving anyone work, instead it goes against the provide and inject pattern.
Each sub field/display can access stores and the api and emit themselves IF they need it, they don't need to start sending around a context ;)

Reply:
Again, that's Vue-only. For the vanilla use case, injection doesn't work.